arcade/pymunk_physics_engine.py

```diff
@@ -70,21 +70,21 @@ def add_sprite(self,
                    ):
         """ Add a sprite to the physics engine.
 
-            :param sprite: The sprite to add
-            :param mass: The mass of the object. Defaults to 1
-            :param friction: The friction the object has. Defaults to 0.2
+            :param sprite: The sprite to add.
+            :param mass: The mass of the object. Defaults to 1.
+            :param friction: The friction the object has. Defaults to 0.2.
             :param elasticity: How bouncy this object is. 0 is no bounce. Values of 1.0 and higher may behave badly.
             :param moment_of_inertia: The moment of inertia, or force needed to change angular momentum. \
             Providing infinite makes this object stuck in its rotation.
             :param body_type: The type of the body. Defaults to Dynamic, meaning, the body can move, rotate etc. \
             Providing STATIC makes it fixed to the world.
-            :param damping: See class docs
-            :param gravity: See class docs
+            :param damping: See class docs.
+            :param gravity: See class docs.
             :param max_velocity: The maximum velocity of the object.
-            :param max_horizontal_velocity: maximum velocity on the x axis
-            :param max_vertical_velocity: maximum velocity on the y axis
-            :param radius:
-            :param collision_type:
+            :param max_horizontal_velocity: Maximum velocity on the x axis.
+            :param max_vertical_velocity: Maximum velocity on the y axis.
+            :param radius: Radius for the shape created for the sprite.
```

Comment:
What's the unit on this? Pixels?

Reply:
It should be doesn't makes sens to use any other unit.

Reply:
I'd add that to the docstring.

Reply:
I mean if you say so, I will add it. There is no harm in explicitly clearing something. Also if you had the question others will too, so I will add it.

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1642/commits/fed3d1dece977cd4aaa544db4797a5ec62feb310